css/index.css

```diff
@@ -1,3 +1,432 @@
+}
+
+.main-header .top-image-container {
+    align-items: flex-end;
+	width: 49%;
+}
+
+.main-header img {
+	display: flex;
+    margin-left: 75px;
+    align-self: flex-end;
+
+}
+
+.main-header .big-text {
+	display: flex;
+	justify-content: flex-end;
+    width: 49%;
+    margin-bottom: 30px;
+    flex-wrap: wrap;
+}
+
+.main-header h1 {
+    font-size: 74px;
+    display: flex;
+    align-self: flex-end;
+}
+
+.main-header .get-started-container {
+	order: 2;
+	justify-content: center;
+	display: flex;
+    width:50%;
+    margin-right: 50px;
+    margin-bottom: 15px;
+}
+
+.main-header p {
+	border: 1px solid black;
+    padding: 5px 45px;
+    align-self: center;
+    justify-content: flex-start;
+}
+
+
+
+.main-content1 {
+	margin: 20px 0;
+	display: flex;
+	width: 100%;
+}
+
+
+.main-content1 h2 {
+    margin: 10px 0;
+    
+}
+
+.main-content1 p {
+	margin: 10px 0;
+	width: 100%;
+}
+
+/* Don't need styles for these, but leaving them just in case */
+
+/* .main-content1 .features {
+
+}
+
+.main-content1 .about {
+	
+} */
+
+.middle-image img{
+	margin: 20px auto;
+	width: 100%;
+}
+
+.main-content2 {
+	display: flex;
+	width: 100%;
+	margin: 10px 0;
+    height: 200px;
+}
+
+.main-content2 .services {
+	text-align: left;
+	display: flex;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 .product {
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 .vision {
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 h2 {
+	margin-bottom: 20px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+}
+
+.main-content2 p {
+	display: flex;
+	margin: 2px;
+	width: 100%;
+}
+
+.contact {
+	margin: 20px 0;
+}
+
+.footer {
+	margin: 20px;
+	padding: 20px;
+	text-align: center;
+}
+
+
+/* Services Page  */
+
+.servicespage .top-image {
+	margin-top: 37px;
+	width: 100%;
+}
+
+.servicespage .services {
+	height: 100px;
+	width: 900px;
+	margin-top: 15px;
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.servicespage .box {
+	width: 435px;
+	height: 200px;
+	border: 1px solid black;
+	margin-top: 30px;
+	padding: 10px;
+	background: whitesmoke;
+}
+
+.servicespage .main-content {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+
+.servicespage .main-content .learn-more {
+	background: white; 
+	margin: 25px 0;
+	width: 150px;
+	border-radius: 10%;
+	box-shadow: remove;
+	border-style: solid;
+	border: 1px solid black;
+	height: 35px;
+}
+.servicespage .bottom-section {
+	display: flex;
+	margin-top: 40px;
+}
+
+.servicespage .some-facts {
+	margin: 20px 0;
+	width: 450px;
+	margin-right: 40px;
+}
+
+.servicespage .some-facts li {
+	list-style: disc;
+	margin: 10px 20px;
+}
+
+.servicespage footer {
+	text-align: center;
+	margin: 50px;
+}
+
+@media (max-width: 800px) {
+    .main-header img {
+        display: none;
+    }
+
+    .container {
```

Comment:
Container should be 100%, if you're doing 95% it won't change the layout until you're 95% of 800px.